modernpython/cimdatatype_header.py

```diff
@@ -0,0 +1,3 @@
+from ..utils.datatypes import CIMDatatype
+from ..utils.profile import Profile
+from .enum import *
```

Comment:
I understand that it will not break anything because you are in control of `.enum`, but I would still either suggest having explicit import, or maybe just use eg. `enum.Integer` in the resource files, which I think has my preference (question of good practice/least surprise - it's tricky to directly find eg. Integer from a * import)

Reply:
here it is easy, because the enum used are limited, it is a bit more complex in other cases, with the 1 file for all resources approach, clearly it is easier to compute required imports in the 1 resource 1 file.

Reply:
the header includes now only the two Enum used